src/ape/managers/project.py

```diff
@@ -228,6 +228,19 @@ def extract_manifest(self) -> PackageManifest:
             return self._extract_local_manifest(temp_project_path)
 
 
+class PackageManifestWrapper:
```

Comment:
Could this just be `Package`?

Reply:
Ah, I deleted this class and went with a different approach that didn't involve adding a new class.

```python
def dependencies(self) -> Dict[str, DependencyAPI]:
```

Reply:
okay hahaha, will check it out